torch/optim/lr_scheduler.py

```diff
@@ -1367,6 +1367,8 @@ def __init__(self, optimizer, T_0, T_mult=1, eta_min=0, last_epoch=-1, verbose=F
             raise ValueError("Expected positive integer T_0, but got {}".format(T_0))
         if T_mult < 1 or not isinstance(T_mult, int):
             raise ValueError("Expected integer T_mult >= 1, but got {}".format(T_mult))
+        if not isinstance(eta_min, float):
+            raise ValueError("Exprected float, but got {}".format(eta_min))
```

Comment:
```suggestion
            raise ValueError("Expected float eta_min, but got {} of type {}".format(eta_min, type(eta_min))
```
nit with spelling, and give a better error message if the type is string since it’d be hard to tell without printing the type if someone passes in “1e-5”.

Reply:
Ah! Sorry for the typo. Thanks for having `type` in the error message. This makes more sense. @janeyx99